src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/AddArtworksModal.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react"
+import { Flex, Text, ModalDialog, Spacer, Button } from "@artsy/palette"
+import { SortFilter } from "Components/SortFilter"
+import { ArtworksList, TEMP_ARTWORKS } from "./ArtworksList"
+import { useTranslation } from "react-i18next"
+
+interface AddArtworksModalProps {
+  visible: boolean
+  onClose: () => void
+  onComplete: () => void
+  listName: string
+}
+
+const AddArtworksModal: React.FC<AddArtworksModalProps> = ({
+  listName,
+  onClose,
+}) => {
+  const { t } = useTranslation()
+
+  const handleSave = () => {
+    onClose()
+  }
+
+  return (
+    <ModalDialog
+      m={["0", "auto"]}
+      dialogProps={{
+        width: ["100%", 700],
+        height: ["100%", "auto"],
+        maxHeight: [null, 800],
+      }}
+      onClose={onClose}
+      title={t("collectorSaves.addArtworksModal.modalTitle", {
+        value: listName,
+      })}
+      data-testid="AddArtworksModal"
```

Comment:
@gkartalis as far as I remember, is it always better to avoid using `data-testid`? is that right?

Reply:
depends 🤷  

I know that there are the following references in force:
- `data-test-id`
- `data-testid` 
- `data-test`

The integrity [docs](https://github.com/artsy/integrity#selecting-specific-elements) suggest to use `data-test` for e2e tests (I am guessing something similar should be the case for acceptance tests in force).

Also [`@testing-library/react`](https://testing-library.com/docs/queries/bytestid/) uses `data-testid` for the `byTestId` query.

So that makes me think that we should avoid the `data-test-id` 🤔

Reply:
gotcha, thanks!